src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -21,6 +21,26 @@ const CountdownTimer = () => {
   //   circlePath.style.animation = `countDown ${settings.workInterval * 60} linear`
   // }
 
+  const displayNotification = () => {
+    if(Notification.permission == 'granted') {
+      navigator.serviceWorker.getRegistration().then(reg => {
```

Comment:
Looking forward to customizing this notification!

Reply:
We gotta find a way to let the people know that Som Timer Cares™ every time.